src/main/java/seedu/address/storage/XmlAdaptedAppointment.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.storage;
+
+import javax.xml.bind.annotation.XmlElement;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+import seedu.address.model.appointment.Appointment;
+import seedu.address.model.appointment.Type;
+
+/**
+ * JAXB-friendly adapted version of the Appointment
+ */
+
+@XmlRootElement
+public class XmlAdaptedAppointment {
+
+    @XmlElement(required = true)
+    private Type type;
```

Comment:
You'll need to change this to an XmlAdapted equivalent, else the XML representation of your object will not store Type data.